### ISSUE TITLE
Would you like a new maintainer?

### ISSUE THREAD

**Issue by LunarGem** [Lamikins/HabitiFit#11]
I noticed you mentioned you weren't maintaining this anymore. Would you like someone else to maintain it for you? I could probably attempt to do that, or maybe ask on the Trello or something for volunteers.

**Comment by Lamikins** [comment]
Yes, that would be great! Let me know when they are ready to assume ownership.

**Comment by benmadsen** [comment]
I went ahead and forked this project in the hopes of getting it it updated.  I take it this does not take advantage of FitBit API Subscriptions?

**Comment by dariuslam** [comment]
Hi ben, no it does not.  Good luck!

**Comment by KessaTurnbull** [comment]
Can I know if there was any progress on the new maintainer?

**Comment by believe2acheive** [comment]
Please let me know if I can help in any way. I'm speaking with several other people who would be open to helping.
